views/get-started.erb

```diff
@@ -39,10 +39,6 @@
 				When you <a href="/signup">request an account</a> you can invite as many team members as you need and choose the ones who’ll be in charge of managing users. They’ll be able to request more user accounts, including ones for external suppliers, and assign <a href="https://docs.cloud.service.gov.uk/#user-roles">roles and permissions</a>.
 			</p>
 
-			<p>
-				If you already have an account please refer to our <a href="/next-steps">Start using GOV.UK PaaS</a> page.
```

Comment:
Should this line remain but instead redirect to the admin app?

Reply:
Ah yeah, asked both @paroxp and @chrisfarms about that and they thought that it would be better if it was removed.  I'm not sure why it's there in the first place though and if we have user research suggesting it should be there, we could test again though and see if it causes problems not being there as the user journey will probably be different with the new admin app being there, as it originally pointed to something that wasn't very interactive.